src/spdx_tools/spdx3/writer/json_ld/context.json

```diff
@@ -264,7 +278,7 @@
         "@type": "xsd:anyURI"
     },
     "locator": {
-        "@id": "core:locator",
+        "@id": "security:locator",
```

Comment:
There are two properties named `locator` in the spec, one in `core`, one in `security`. We can expand either one of them (in that case I would go with `core`) or neither (i.e. keep using `core:locator`and `security:locator` everywhere).

Reply:
I've added a check to prefer and keep core properties here.